pyproject.toml

```diff
@@ -115,7 +128,7 @@ omit = [
 
 # set dynamic versioning capabilities for project
 [tool.poetry-dynamic-versioning]
-enable = true
+enable = false
```

Comment:
Since this is now set to false, does the comment need to change to reflect this?

Reply:
Thanks for catching this! I'm not exactly sure how this got changed, yes it should be changed back.